.eslintrc

```diff
@@ -56,7 +56,9 @@
     "@emotion/syntax-preference": [
       "error",
       "string"
-    ]
+    ],
+    "@typescript-eslint/no-unsafe-return": "off",
```

Comment:
Why ?

Reply:
ESLint would throw this error on some tests, I don't really understand why :(

![Screenshot 2022-05-04 at 16 14 56](https://user-images.githubusercontent.com/43268759/166700419-6f39878a-181c-4f54-be75-8a0a5cc9e96c.png)

Reply:
Fixed by installing the types of `@testing-library/jest-dom` in 178b511316b015b34f50c3bee0b542a7f9fa2e94, see https://github.com/testing-library/jest-dom/#installation